lib/jsdom/living/nodes/HTMLAnchorElement-impl.js

```diff
@@ -2,14 +2,41 @@
 const idlUtils = require("../generated/utils");
 const HTMLElementImpl = require("./HTMLElement-impl").implementation;
 const HTMLHyperlinkElementUtilsImpl = require("./HTMLHyperlinkElementUtils-impl").implementation;
+const {navigate} = require("../window/navigation");
+const isDisabled = require("../helpers/form-controls").isDisabled;
 
+// https://html.spec.whatwg.org/#the-a-element
 class HTMLAnchorElementImpl extends HTMLElementImpl {
   constructor(args, privateData) {
     super(args, privateData);
 
     this._htmlHyperlinkElementUtilsSetup();
   }
 
+  _activationBehavior() {
+    if (this.hasAttribute("disabled")) {
+      return;
+    }
+
+    if (!this.href || !this.url) {
+      return;
+    }
```

Comment:
This is the "otherwise" in https://html.spec.whatwg.org/#following-hyperlinks-2 point 10

It gives browsers a lot of room to do whatever they like, but one of the options is "do nothing" and that seems to be what chrome does.